### ISSUE TITLE
Can I please ask you to fix your code!

### ISSUE THREAD

**Issue by TechSide1558** [nerdaxic/glados-voice-assistant#28]
It is saying I need to install so many flipping files. I CANNOT install torch. I am doing this on a raspberry pi using the RASPBERRY PI branch.

**Comment by eternalliving** [comment]
I don't think you need torch for the RPi version as you cannot create the glados voice responses on the RPi itself and torch is only used by the text-to-speech engine. You can run the TTS engine on another computer, or select one of the online servers to fetch the speech from.

**Comment by TechSide1558** [comment]
How on earth do you do that? It is asking me to use torch.

**Comment by eternalliving** [comment]
The Raspberry Pi branch should be set up to use https://glados.c-net.org/ to collect voice samples. If you want to set the engine up on another machine you'll have to look at the [Glados-TTS engine](https://github.com/nerdaxic/glados-tts/tree/c645589b88254f14d339203c2bfa06212e4957f5) repo.

Can you tell me what is asking you to use torch? Is it during install or are you able to run glados.py? What file seems to be calling it? Double check you are using the Raspberry Pi branch.

**Comment by TechSide1558** [comment]
I tried to run the tts on a windows 11 laptop, but i CANNOT install torch on windows.

**Comment by eternalliving** [comment]
Make sure you have installed pyTorch with pip and not Conda. Try `pip show torch` and see if that yeilds any results. If it doesn't, then you have installed the wrong version of pyTorch.

**Comment by TechSide1558** [comment]
I used `pip3 install torch torchvision torchaudio` and it did not work

**Comment by eternalliving** [comment]
Sadly that is a pyTorch issue. I would suggest doing some online searches on how to install pytorch for windows. Good luck with it and hopefully someone can help you get Torch installed. 

If you have any problems after that feel free to raise another issue, but this currently has nothing to do with the glados code itself.

**Comment by TechSide1558** [comment]
I tried researching, but I cannot find anything. Also, how do you use the [https://glados.c-net.org/](url) to process it. It just says the speech synthesis core is offline when I use it.

**Comment by eternalliving** [comment]
You couldn't find anything on installing pyTorch on windows 11? You could try this [link](https://www.geeksforgeeks.org/install-pytorch-on-windows/), although I installed it manually through the pyTorch website so not sure of it's accuracy.

Also, to get glados.c-net.org to process you'll have to change your code a bit. As the Raspberry branch was left behind before converting all the settings to yaml, the speech engine was hard coded. The one that is currently there was a test site for the glados-tts engine while it was in development.

in gladosTTS.py, uncomment the following line (line 61)
```
TTSCommand = 'curl -L --retry 30 --get --fail --data-urlencode "text='+cleanTTSLine(line)+'" -o "'+synthFolder+cleanTTSFile(line)+'" "https://glados.c-net.org/generate"'
```

and add a comment (or remove) the following lines (line 64 & 65)
```
text = urllib.parse.quote(cleanTTSLine(line))
TTSCommand = 'curl -L --retry 5 --get --fail -o '+synthFolder+cleanTTSFile(line)+' https://glados.2022.us/synthesize/'+text
```
